spiceaidocs/docs/components/data-connectors/duckdb.md

```diff
@@ -18,13 +18,53 @@ datasets:
 
 ## Configuration
 
+### `from`
+
+The `from` field supports one of two forms:
+
+| `from`                         | Description                                                                                                                                             |
+| ------------------------------ | ------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `duckdb:database.schema.table` | Read data from a table named `database.schema.table` in the DuckDB file                                                                                |
+| `duckdb:read_*()`              | Read data using one of the common [data import](https://duckdb.org/docs/data/overview) DuckDB functions, e.g `read_json`, `read_parquet` or `read_csv`. |
+
```

Comment:
DuckDB functions are not restricted to read_*(), functions like glob('*') can also be passed in, e.g.
```YAML
  - from: duckdb:glob('*')
    name: hits
```

Reply:
Yes, any DuckDB functions that create a table/result set can be used

Reply:
Also I learned something new about the product